frontend/src/components/Header/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from "./Header.module.scss";
+const Header=()=>
```

Comment:
Keep the header static. It should be visible on all scroll positions as in design
![Screenshot from 2021-05-23 00-33-52](https://user-images.githubusercontent.com/51409512/119238165-91100c00-bb5e-11eb-81b6-301618074c7c.png)

Reply:
also the brand image on navbar seems to be of low quality, download image from here https://cinemaclub.iitr.ac.in/images/logo.png